modules-magic-drc/pdk_subset/libs.tech/magic/sky130A.tcl

```diff
@@ -0,0 +1,6998 @@
+# Copyright 2020 R. Timothy Edwards
```

Comment:
This file seems super huge, should it be downloaded from somewhere rather than directly included here?